mappings/net/minecraft/item/ArrowItem.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS azz net/minecraft/item/ArrowItem
 	METHOD <init> (Lbbw$a;)V
 		ARG 1 settings
-	METHOD a createProjectile (Lbhi;Lbcb;Lair;)Lawc;
+	METHOD a createArrow (Lbhi;Lbcb;Lair;)Lawc;
 		ARG 1 world
 		ARG 2 stack
-		ARG 3 livingShooter
+		ARG 3 entity
```

Comment:
Isn't that less descriptive than previously ?

Reply:
Another case of my initial generalized name pass. I will rename this to `shooter`.

Reply:
Resolved by 07f735c